urbanpy/download/download.py

```diff
@@ -316,5 +323,71 @@ def osmnx_graph(download_type, network_type='drive', query_str=None,
         if distance is None and download_type == 'point':
             print('Please provide a distance buffer for the point download')
 
-        if geom is None and distance is not None:
-            print('Please provide a Point geometry.')
+        if geom is None and distance is not None: print('Please provide a Point geometry.')
+
+
+
+def search_hdx_dataset(country, repository="high-resolution-population-density-maps-demographic-estimates"):
```

Comment:
Future concern: Is this function necessary? Maybe these lines could be part of `download_hdx_dataset`